hummingbot/market/ddex/ddex_in_flight_order.pyx

```diff
@@ -63,6 +63,13 @@ cdef class DDEXInFlightOrder(InFlightOrderBase):
     def is_cancelled(self) -> bool:
         return self.last_state in {"canceled"}
 
+    @property
+    def is_failure(self) -> bool:
+        # Does not have any 'expiry" or "rejected" order statuses
+        # Ref: https://docs.ddex.io/#place-order-synchronously
+        # Currently not in use
+        return self.last_state in {"partially_filled", "canceled"}
```

Comment:
why is partially_filled a failure condition?
also looks like it's partial_filled

![image](https://user-images.githubusercontent.com/26056161/67048105-70611400-f0e8-11e9-96b7-855df641256a.png)